x-pack/plugins/cases/public/components/case_action_bar/action_bar_status_item.test.tsx

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { mount } from 'enzyme';
```

Comment:
Could you please use RTL (react testing library) for the tests? We are trying to migrate away from `enzyme` and all new tests should be written with RTL. Thanks!